R/mod_08_geo_ch_flag_drug.R

```diff
@@ -595,11 +618,24 @@ mod_08_geo_ch_flag_drug_server <- function(id, export_data) {
           borderless = FALSE,
           columns = list(
             .selection = reactable::colDef(width = 15),
-            `20/21` = reactable::colDef(width = 70, format = reactable::colFormat(digits = 2)),
-            `21/22` = reactable::colDef(width = 70, format = reactable::colFormat(digits = 2)),
-            `22/23` = reactable::colDef(width = 70, format = reactable::colFormat(digits = 2))
+            `20/21` = reactable::colDef(width = 70),
+            `21/22` = reactable::colDef(width = 70),
+            `22/23` = reactable::colDef(width = 70)
+          ),
+          defaultColDef = reactable::colDef(
+            headerClass = "my-header",
+            format = reactable::colFormat(digits = 2),
+            cell = function(val, row, col_name) {
+              if (col_name %in% c(names(geographies), ".selection")) return (val)
+              
+              return (
+                scales::label_comma(
+                  accuracy = .01,
+                  scale_cut = scales::cut_long_scale()
+                )(val)
```

Comment:
Custom format for metric value cells; `scales::label_comma` returns a function, in this case one that rounds to 2 d.p. and uses K, M, B suffixes appropriately.